tenancy-api-mapping/Makefile

```diff
@@ -109,57 +119,55 @@ openapi-lint: node_modules
 		--skip-rule=info-license \
 		--skip-rule=tag-description
 
-#### Help Target ####
-help: ## Print help for each target
-	@echo $(PROJECT_NAME) make targets
-	@echo "Target               Makefile:Line    Description"
-	@echo "-------------------- ---------------- -----------------------------------------"
-	@grep -H -n '^[[:alnum:]_-]*:.* ##' $(MAKEFILE_LIST) \
-    | sort -t ":" -k 3 \
-    | awk 'BEGIN  {FS=":"}; {sub(".* ## ", "", $$4)}; {printf "%-20s %-16s %s\n", $$3, $$1 ":" $$2, $$4};'
+### git checkout targets ###
+
+$(REPO_DIR):
+	@mkdir -p $@
+
+# Get list of directories to checkout repos, named per api mapping
+REPO_LIST := $(shell ls $(API_MAPPING_CONFIG_DIR))
+REPOS := $(addprefix $(REPO_DIR)/, $(basename $(REPO_LIST)))
+
+checkout-repos: $(REPOS) ## checkout all repos
```

Comment:
I think this entire process can be further simplified by using git-submodules instead of maintaining logic to clone repos, but if it works I'm fine with it

Reply:
I removed git submodules because the git tag is specified in the `apimappingconfigcrs/*.yaml`, and changing the tag would also mean changing the submodule commit hash, which would need other logic to keep in sync.

What I've done here is a simpler solution that only uses the tag to clone, which is similar to the previous submodule-based system but that one didn't check in the submodule files and was problematic in a few other ways.

(This all assumes that tag -> hash should not change, which ideally they should not)